prov/shm/src/smr_init.c

```diff
@@ -91,6 +132,10 @@ static int smr_getinfo(uint32_t version, const char *node, const char *service,
 	if (ret)
 		return ret;
 
+	ret = smr_shm_space_check((*info)->tx_attr->size, (*info)->rx_attr->size);
+	if (ret)
+		return ret;
+
```

Comment:
I think we need to clear the entries from info that util_getinfo allocated

Reply:
Thank you. Fixed.